mds/apis/agency_api/v0_x/vehicles.py

```diff
@@ -92,9 +94,15 @@ class DeviceRegisterSerializer(serializers.Serializer):
     def create(self, validated_data):
         provider_id = self.context["request"].user.provider_id
         try:
-            return models.Device.objects.create(
+            device = models.Device.objects.create(
                 provider_id=provider_id, **validated_data
             )
+            models.EventRecord.objects.create(
+                timestamp=datetime.datetime.now(datetime.timezone.utc),
```

Comment:
```suggestion
                timestamp=timezone.now(),
```
Along with `from django.utils import timezone`.

Sorry, I missed that in my review.